src/components/base/outline-accordion/outline-accordion.stories.ts

```diff
@@ -12,35 +12,35 @@ export default {
         code: `
   <outline-accordion label=label clean=clean single-panel=singlePanel>
     <outline-accordion-panel slot="panels">
-      <h5 slot="heading">Heading 1</h5>
+      <span slot="heading">Heading 1</span>
```

Comment:
Let's remove all the "Heading 1", 2, etc. for a more generic title like "Accordion Item 1" sine we're reducing this to a span tag.